### ISSUE TITLE
Status always "null"

### ISSUE THREAD

**Issue by TxHawks** [MercymeIlya/last-workflow-status#6]
With the following:
```yaml
jobs:
  prepare:
    outputs:
      prevStatus: ${{ steps.prevStatus.outputs.last_status }}
    runs-on: ubuntu-latest
    steps:
      - name: Get status of last workflow
        uses: MercymeIlya/last-workflow-status@v0.3
        id: prevStatus
        with:
          - github_token: ${{ secrets.GITHUB_TOKEN }}
```

`steps.prevStatus.outputs.last_status` is always `null`.

Not sure what I'm doing wrong

**Comment by MercymeIlya** [comment]
@TxHawks can you please show the output of this step?
At least https://api.github.com/ link with filled parameters.

maybe it was the first commit in new branch, that's why there was no previous workflow as well as conclusion for it.

**Comment by jbaiza** [comment]
According to https://michaelcurrin.github.io/dev-cheatsheets/cheatsheets/ci-cd/github-actions/persist.html, the last line (https://github.com/MercymeIlya/last-workflow-status/blob/master/action.yml#L46) should be without curly braces:
`echo "::set-output name=last_status::$last_status"`
I created #12 with a fix.
edit:
It seems that the main problem was with replacing `/` in the branch name. Created a fix - #13.
